test/Google.Api.Gax.Grpc.Tests/RetryTest.cs

```diff
@@ -360,7 +359,7 @@ public AsyncServerStreamingCall<SimpleResponse> ServerStreamingMethodSync(Simple
                     }
                     return Metadata.Empty;
                 });
-                var response = (AsyncServerStreamingCall<SimpleResponse>)ctor.Invoke(new object[] { responseStream, responseHeaders, null, null, null });
+                var response = new  AsyncServerStreamingCall<SimpleResponse>(responseStream, responseHeaders, null, null, null);
```

Comment:
nit: Remove extra space after `new`

Reply:
Done; rebasing and force pushing. Will merge on green.